jwt.test-d.ts

```diff
@@ -55,22 +55,31 @@ expectAssignable<Function>(app.jwt.verify)
 expectAssignable<Function>(app.jwt.decode)
 
 app.addHook("preHandler", async (request, reply) => {
-    // assert request and reply specific interface merges
-    expectAssignable<Function>(request.jwtVerify)
-    expectAssignable<object | string | Buffer>(request.user)
-    expectAssignable<Function>(reply.jwtSign)
+  // assert request and reply specific interface merges
+  expectAssignable<Function>(request.jwtVerify)
+  expectAssignable<object | string | Buffer>(request.user)
+  expectAssignable<Function>(reply.jwtSign)
 
-    try {
-        await request.jwtVerify();
-    }
-    catch (err) {
-        reply.send(err);
-    }
+  try {
+    await request.jwtVerify();
+  }
+  catch (err) {
+    reply.send(err);
+  }
 });
 
 app.post('/signup', async (req, reply) => {
-    const token = app.jwt.sign({ user: "userName" });
-    let data = await app.jwt.verify(token);
-    const user = req.user;
-    reply.send({ token });
+  const token = app.jwt.sign({ user: "userName" });
+  let data = await app.jwt.verify(token);
+  const user = req.user;
+  reply.send({ token });
 });
+
+// define custom payload

```

Comment:
could you create a new testing file that has some declaration merging and tests for it?

Reply:
Declaring the module will pollute the global type environment, I have not found a good way to use dts to test

Reply:
Do we really care about adding types to the global type env? I think we should test this properly.

Reply:
Good point, this is good to go then